alsaaudio.c

```diff
@@ -827,6 +827,25 @@ they represent values stored by pyalsaaudio and they are prefixed with ' (call v
 ");
 
 
+static PyObject *
+alsa_asoundlib_version(PyObject * module, PyObject *args)
+{
+	PyObject *value;
+
+	if (!PyArg_ParseTuple(args,":asoundlib_version"))
+		return NULL;
+
+	value=PyUnicode_FromString(snd_asoundlib_version());
+
+	return value;
+}
+
+PyDoc_STRVAR(asoundlib_version_doc,
+"asoundlib_version() -> string\n\
+\n\
+Returns ALSA version string. \n\
+");
+
 static PyObject *
```

Comment:
Would you mind extending the documentation in doc/libalsaaudio.rst, too?

Reply:
I added entries for the new functions. I tried to adapt to conventions used.